qt_gui_app/scripts/qt_gui_app

```diff
@@ -3,12 +3,23 @@
 import os
 import sys
 
+<<<<<<< HEAD
```

Comment:
Uh oh!

Reply:
Good catch. How did git let me miss this?

Reply:
Resolved